web/components/UserProfile.ce.vue

```diff
@@ -136,11 +137,16 @@ onMounted(() => {
 
       <NotificationsSidebar />
 
-      <UpcDropdownMenu :t="t">
+      <DropdownMenu align="end" side="bottom" :side-offset="4">
         <template #trigger>
           <UpcDropdownTrigger :t="t" />
         </template>
-      </UpcDropdownMenu>
+        <template #content>
+          <div class="w-[350px]">
```

Comment:
`w-[350px]` will need a refactor to work better responsively.

Please try this out on a `7.2.0-beta.x.x` test release for a better idea of what you'll need to do.

Probably will need a min-width for "non-mobile" sizes in addition to a max width.

I don't know if something like `max-w-[350px] sm:min-w-[350px]` or something would do the trick.